FrontEndLib/ListBoxWidget.cpp

```diff
@@ -660,6 +660,36 @@ bool CListBoxWidget::RemoveItem(const void* pKey)
 	return bRemoved;
 }
 
+bool CListBoxWidget::RemoveItems(const CIDSet* pKeys)
```

Comment:
Suggest passing in as a const ref (i.e., "const CIDSet& keys").  Just as fast as a pointer and cleaner syntax.